src/assets/stylesheets/main.scss

```diff
@@ -11,6 +11,7 @@ body {
 }
 
 a {
+  display: block;
```

Comment:
This does not seem like a good idea 🤓
How about the cases if we would want to have a link within a text paragraph? 👀